Shared/mods/deathmatch/logic/lua/LuaBasic.h

```diff
@@ -123,7 +123,7 @@ namespace lua
     template <typename T>
     std::enable_if_t<std::is_class_v<T>, int> Push(lua_State* L, T* val)
     {
-        lua_pushelement(L, val);
+        lua_pushuserdata(L, val);
```

Comment:
pushuserdata is way slower.
```cpp
if constexpr (std::is_base_of_v<CElement, T>)
    lua_pushelement(L, val);
else
    lua_pushuserdata(L, val);
```

Reply:
Yeah I thought about that too. But since I'll redo all of that with concepts anyway, I just skipped over it for now and kept the fix small.

Reply:
Actually I'll change this up... `lua::Push` won't change too much with concepts